docs/plugins/graphback-client.md

```diff
@@ -49,7 +48,7 @@ Plugin allows to create files in many formats to suit the needs of your applicat
   fragmentOnly?: boolean
 
   /**
-   * RelativePath for the output files created by generator
+   * RelativePath for the output file created by generator
    */
   outputPath: string
```

Comment:
Output file then?

Reply:
Will change `outputPath` to `outputFile`